components/locale-chooser.js

```diff
@@ -121,48 +188,57 @@ function localeToHref(locale) {
   return 'https://github.com/kentcdodds/glamorous-website/blob/master/other/CONTRIBUTING_DOCUMENTATION.md'
 }
 
-function getLocaleAndHost() {
-  const locale = process.env.LOCALE
+function getHost(currentLocale) {
   const {host} = window.location
-  // eslint-disable-next-line no-unused-vars
   const [localePart, ...rest] = host.split('.')
-  if (supportedLocales.includes(locale) || localePart !== locale) {
-    return {locale, host: rest.join('.')}
-  } else {
-    return {locale: fallbackLocale, host}
+  if (localePart === currentLocale) {
+    return rest.join('.')
   }
+
+  return host
 }
 
-function mapLocale(key = fallbackLocale) {
-  const localeMap = {
-    en: {
-      key,
-      display: 'English',
-      Flag: EnSvg,
-    },
-    es: {
-      key,
-      display: 'Español',
-      Flag: EsSvg,
-    },
-    fr: {
-      key,
-      display: 'Français',
-      Flag: FrSvg,
-    },
-    de: {
-      key,
-      display: 'Deutsche',
-      Flag: DeSvg,
-    },
-    help: {
-      key,
-      display: content.help,
-    },
+// eslint-disable-next-line complexity
+function mapLocale(key) {
+  const options = {width: '1em', height: '100%'}
+  switch (key) {
+    case 'en':
+      return {
+        key,
+        display: 'English',
+        Flag: <EnSvg {...options} />,
```

Comment:
Could we leave most of this stuff the way it was? I liked the way the rendering of `Flag` better the way it was before.